AlphaWallet/Market/UniversalLinkHandler.swift

```diff
@@ -113,17 +126,9 @@ public class UniversalLinkHandler {
     }
     
     func getTokenIdsFromSpawnableLink(linkBytes: [UInt8]) -> [BigUInt] {
-        let tokens = linkBytes[84..<linkBytes.count]
-        let tokenCount = tokens.count / 32
-        var tokenIds = [BigUInt]()
-        var startPos: Int = 0
-        for i in 0..<tokenCount {
-            let tokenId: [UInt8] = Array(tokens[startPos...32 * i])
-            let tokenIdBigUInt = BigUInt(Data(bytes: tokenId))
-            tokenIds.append(tokenIdBigUInt)
-            startPos += 32
-        }
-        return tokenIds
+        let bytes = linkBytes[84..<linkBytes.count]
+        let tokenIds = bytes.split(separator: 32)
```

Comment:
Nice! `let bytes...` is much simpler.

These 2 are different though:

    let tokens = bytes.split(intoChunksOf: 32)

    let tokenIds = bytes.split(separator: 32)

The former breaks the array into subarrays, each with length 32, the latter breaks the array into subarrays too, but by using the value `32` as the separator.